aliSimple.go

```diff
@@ -189,7 +189,9 @@ func urlEncode(ins string) string {
 	str0 = strings.Replace(str0, "+", "%20", -1)
 	str0 = strings.Replace(str0, "*", "%2A", -1)
 	str0 = strings.Replace(str0, "%7E", "~", -1)
+
 	str0 = url.QueryEscape(str0)
+	str0 = strings.Replace(str0, "%26", "&", -1)
```

Comment:
in sample should not escape "&"

https://help.aliyun.com/document_detail/29747.html#:~:text=3Dtestid%26Action%253DDescribeDomainRecords-,%26,-DomainName%253Dexample.com